website/content/en/preview/getting-started/_index.md

```diff
@@ -13,3 +13,241 @@ Learn more about Karpenter and how to get started below.
+
+* coredns
+* metric-server
+
+You can edit them with `kubectl edit deploy ...` and you should add node affinity for your static node group instances.
+Modify the value to match your `$NODEGROUP`.
+
+```
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: eks.amazonaws.com/nodegroup
+                operator: In
+                values:
+                - ${NODEGROUP}
+```
+
+## Remove Cluster Autoscaler
+
+Now that Karpenter is running, if your cluster is running cluster autoscaler you have the option of removing it.
+To do that, scale the number of replicas to zero.
+
+{{% script file="./scripts/step12-scale-cas.sh" language="bash" %}}
+
+To get rid of the instances that were added from the node group we can scale our nodegroup down to a minimum size to support Karpenter and other critical services.
+We suggest a minimum of 2 nodes for the node group.
+
+> Note: If your workloads do not have [pod disruption budgets](https://kubernetes.io/docs/tasks/run-application/configure-pdb/) set,
+> the following command **will cause workloads to be unavailable.**
+
+{{% script file="./scripts/step13-scale-ng.sh" language="bash" %}}
+
+If you have a lot of nodes or workloads you may want to slowly scale down your node groups by a few instances at a time.
+It is recommended to watch the transition carefully for workloads that may not have enough replicas running or disruption budgets configured.
+
+### Verify Karpenter
+
+As nodegroup nodes are drained you can verify that Karpenter is creating nodes for your workloads.
+
+```bash
+kubectl logs -f -n karpenter -c controller -l app.kubernetes.io/name=karpenter
+```
+
+You should also see new nodes created in your cluster as the old nodes are removed
+
+```bash
+kubectl get nodes
+```
+## First Use
+
+Karpenter is now active and ready to begin provisioning nodes.
+Create some pods using a deployment, and watch Karpenter provision nodes in response.
+
+### Automatic Node Provisioning
+
+This deployment uses the [pause image](https://www.ianlewis.org/en/almighty-pause-container) and starts with zero replicas.
+
+{{% script file="./scripts/step13-automatic-node-provisioning.sh" language="bash"%}}
+
+### Automatic Node Termination
+
+Now, delete the deployment. After 30 seconds (`ttlSecondsAfterEmpty`),
+Karpenter should terminate the now empty nodes.
+
+{{% script file="./scripts/step14-deprovisioning.sh" language="bash"%}}
+
+### Manual Node Termination
+
+If you delete a node with kubectl, Karpenter will gracefully cordon, drain,
+and shutdown the corresponding instance. Under the hood, Karpenter adds a
+finalizer to the node object, which blocks deletion until all pods are
+drained and the instance is terminated. Keep in mind, this only works for
+nodes provisioned by Karpenter.
+
+{{% script file="./scripts/step15-delete-node.sh" language="bash"%}}
+
+## Cleanup
```

Comment:
Maybe worth removing this section now since you don't know how the user created the cluster.

Reply:
I'll remove the `eksctl` command for deleting the cluster. I think the other steps would be a good way to remind them how to clean up the Karpenter stuff they added from the guide.

Reply:
I think the other commands work, but I still see the eksctl command there. Can we remove it from the bottom of the file?